mongodb/src/it/scala/monix/connect/mongodb/MongoSinkSuite.scala

```diff
@@ -273,15 +273,16 @@ class MongoSinkSuite extends AsyncFlatSpec with MonixTaskSpec with Fixture with
 
   "updateManyPar" should "update many documents per each received request whereas requests are grouped in lists. " +
     "Requests from a single group are executed at once in parallel." in {
-    val name1 = "Name1"
-    val name2 = "Name2"
-    val name3 = "Name3"
+    val name1 = "Name4"
+    val name2 = "Name5"
+    val name3 = "Name6"
+    val name4 = "Name7"
```

Comment:
Why does the string value does not match with the variable name?
```scala
    val name1 = "Name1"
    val name2 = "Name2"
    val name3 = "Name3"
    val name4 = "Name4"
```

Reply:
Actually, it does not have any purpose. Personally, I'm not a fan of using the same names multiple times in multiple test cases (`updateMany` has names from 1 to 3). But, I'm making a commit with your suggestion applied.

Reply:
Me neither, that's why we should start using generators in combination with prefixes in all our test suites. 
I don't mind if it's 1 2 3 or 4 3 5, but the var name and value should be in-sync